R/modelFitting.R

```diff
@@ -457,69 +458,9 @@ compBayes <- function(dat,
-  # condNum <- getCond(uBio, bio = FALSE)
-  # condNames <- getName(uBio)
-  #
-  # refC <- dat[[1]][1, "tag1"]
-  #
-  # nCond <- length(unique(condNum))
-  # uCond <- unique(c(refC, unique(condNum)))
-  # uCond <- uCond[order(uCond)]
-  #
-  # nProts <- length(unique(condNames))
-  #
-  # indices <- lapply(1:nProts, function(x)
-  #   which(condNames == levels(factor(condNames))[x]))
-  #
-  # condList <- lapply(indices, function(x) condNum[x])
-  #
-  # simpRES <- lapply(indices, function(x)
-  #   alrInv(makeMat(x, model, bio = FALSE, useCov, avgCond = FALSE)))
-  #
-  #
-  # refPos <- which(uCond == refC)
-  # suCond <- uCond[-refPos]
-  # if(length(suCond) == 1){
-  #   lrs <- as.matrix(mapply(function(x, y) x[[2]]$Estimate[match(suCond, y)], simpRES, condList))
-  #   colnames(lrs) <- paste0("Est_avg_fc", suCond)
-  #   lrVars <- as.matrix(mapply(function(x, y) x[[2]]$Variance[match(suCond, y)], simpRES, condList))
-  #   colnames(lrVars) <- paste0("Var_Cond", suCond)
-  #   lrLL95 <- as.matrix(mapply(function(x, y) x[[2]]$LL95[match(suCond, y)], simpRES, condList))
-  #   colnames(lrLL95) <- paste0("LL95_Cond", suCond)
-  #   lrUL95 <- as.matrix(mapply(function(x, y) x[[2]]$UL95[match(suCond, y)], simpRES, condList))
-  #   colnames(lrUL95) <- paste0("UL95_Cond", suCond)
-  # }else{
-  #   lrs <- t(mapply(function(x, y) x[[2]]$Estimate[match(suCond, y)], simpRES, condList))
-  #   colnames(lrs) <- paste0("Est_avg_fc", suCond)
-  #   lrVars <- t(mapply(function(x, y) x[[2]]$Variance[match(suCond, y)], simpRES, condList))
-  #   colnames(lrVars) <- paste0("Var_Cond", suCond)
-  #   lrLL95 <- t(mapply(function(x, y) x[[2]]$LL95[match(suCond, y)], simpRES, condList))
-  #   colnames(lrLL95) <- paste0("LL95_Cond", suCond)
-  #   lrUL95 <- t(mapply(function(x, y) x[[2]]$UL95[match(suCond, y)], simpRES, condList))
-  #   colnames(lrUL95) <- paste0("UL95_Cond", suCond)
-  # }
-  # popLrTab <- data.frame(Protein = levels(factor(condNames)), lrs, lrVars, lrLL95, lrUL95,
-  #                        stringsAsFactors = F)
-  #
-  # #Now get the proportions
-  # lrs <- t(mapply(function(x, y) x[[1]]$Estimate[match(uCond, unique(c(refC, y)))], simpRES, condList))
-  # colnames(lrs) <- paste0("Est_avg_Prop", uCond)
-  # lrVars <- t(mapply(function(x, y) x[[1]]$Variance[match(uCond, unique(c(refC, y)))], simpRES, condList))
-  # colnames(lrVars) <- paste0("Var_Cond", uCond)
-  # lrLL95 <- t(mapply(function(x, y) x[[1]]$LL95[match(uCond, unique(c(refC, y)))], simpRES, condList))
-  # colnames(lrLL95) <- paste0("LL95_Cond", uCond)
-  # lrUL95 <- t(mapply(function(x, y) x[[1]]$UL95[match(uCond, unique(c(refC, y)))], simpRES, condList))
-  # colnames(lrUL95) <- paste0("UL95_Cond", uCond)
-  #
-  # popPTab <- data.frame(Protein = levels(factor(condNames)), lrs, lrVars, lrLL95, lrUL95,
-  #                       stringsAsFactors = F)
-
-
   } #end else (not simple case)
 
-  if(sumPtm > 0){
+  if(sumPtm > 0 & modelN == 2){
```

Comment:
What happens if `modelN != 2`?

Reply:
Stan modeling stops and the program repeatedly returns the message "Kill all humans".